docs/src/opt_cep.md

```diff
@@ -12,41 +14,50 @@ The model is designed to minimize social costs by minimizing the following objec
 min \sum_{account,tech}COST_{account,'EUR/USD',tech} + \sum LL \cdot  cost_{LL} + LE \cdot  cos_{LE}
 ```
 
-## Variables and Sets
+## Sets
 The models scalability is relying on the usage of sets. The elements of the sets are extracted from the input data and scale the different variables. An overview of the sets is provided in the table. Depending on the models configuration the necessary sets are initialized.
 
+The sets are setup as a dictionary and organized as `set[tech_name][tech_group]=[elements...]`, where:
+- `tech_name` is the name of the dimension like e.g. `tech`, or `node`
+- `tech_group` is the name of a group of elements within each dimension like e.g. `["all", "generation"]`. The group `'all'` always contains all elements of the dimension
+- `[elements...]` is the Array with the different elements like `["pv", "wind", "gas"]`
+
 | name             | description                                                           |
 |------------------|-----------------------------------------------------------------------|
 | lines            | transmission lines connecting the nodes                               |
 | nodes            | spacial energy system nodes                                           |
-| tech             | fossil and renewable generation as well as storage technologies       |
+| tech             | generation, conversion, storage, and transmission technologies        |
+| carrier          | carrier that an energy balance is calculated for `electricity`, `hydrogen`...|
 | impact           | impact categories like EUR or USD, CO 2 − eq., ...                    |
 | account          | fixed costs for installation and yearly expenses, variable costs      |
 | infrastruct      | infrastructure status being either new or existing                    |
-| sector           | energy sector like electricity                                        |
 | time K           | numeration of the representative periods                              |
-| time T           | numeration of the time intervals within a period                      |
-| time T e         | numeration of the time steps within a period                          |
-| time I           | numeration of the time invervals of the full input data periods       |
-| time I e         | numeration of the time steps of the full input data periods           |
+| time T period    | numeration of the time intervals within a period                      |
```

Comment:
I do not understand the difference between time point and time interval. Are we talking about T time steps of length `delta t` within a period?